deploy/MIGRATION.md

```diff
@@ -143,6 +153,18 @@ Before proceeding, always check on the cluster page the *Upgrade Insights* tab w
 availability of Kubernetes resources in each version. The following sections omit this check if no resource is
 expected to be reported in the context of a standard deployment performed with the tools in this repository.
 
+#### 1.28 to 1.32
```

Comment:
And here same logic for generic instructions + specific ones for `1.29`, and `1.25` ?